### PR TITLE
MINOR; Update documentation for printing dependencies

### PR DESCRIPTION
**Pull request by jsancio**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ If needed, you can specify the Scala version with `-PscalaVersion=2.13`.
     ./gradlew testJar
 
 ### Determining how transitive dependencies are added ###
```
Comment (ijuma) [inline README.md]: Maybe we should delete this and and move "Dependency Analysis" above "common build options"

Comment (jsancio) [reply README.md]: I agree. I missed that section. Let me update it.

**Comment by jsancio** [comment]
@ijuma What do you think? Can we merge this?

**Review by ijuma — APPROVED** [review]
LGTM, thanks!

**Pull request merged**